### ISSUE TITLE
Review cycle list (radio buttons - 3mo, 6mo etc.) Populates the next review date based on the last reviewed date.

### ISSUE THREAD

**Issue by imperator99** [govau/dta-website-rebuild#53]

**Comment by NoniH** [comment]
Authors should be able to choose from a list of review by periods:
 - 3 months
 - 6 months
 - 12 months
 - 18 Months

This should be a required field when pushing to "published". 

Authors should be able to review a list of all content sorted by closest review-by date (custom report?). 

If possible, alerts are raised with the Author when content passes it's review-by date.

**Comment by imperator99** [comment]
Theoretically, this could be achieved by the following:
1. Users select a review period from a list
2. A calculated field creates a review date based on the last reviewed date + the value of the review period.
3. Rules or similar sends an alert when the review date is reached.

**Comment by imperator99** [comment]
Various modules are not quite ready to handle this in Drupal 8.
I have resolved this by adding a 'review date' field to the relevant content types that defaults to 1 year from the current date when editing content. We can investigate a better solution as it becomes available.
I have also added a view on the 'Content' page that shows all pages sorted by revision date.
Resolved by https://github.com/govau/dta-website-rebuild/commit/abc0fc983ac7bdaec0ef7221d516b15d597a47d0.
